### ISSUE TITLE
setcounter problem

### ISSUE THREAD

**Issue by aisejohan** [pbelmans/gerby-website#158]
See [this page](https://stacks.math.columbia.edu/tag/0FNG) where the second list of items should start with number 10. In the latex we have
 `\begin{enumerate}
\setcounter{enumi}{9}
`

**Comment by pbelmans** [comment]
Good catch! This shouldn't be too hard to implement, I'll try after the weekend.

**Comment by pbelmans** [comment]
Mmm, this seems more challenging than I hoped. Not impossible, just more than an hour's worth of work to get it right. HTML and LaTeX deal with these things differently. Sorry!

A workaround would be to use `\item[(10)]` for the time being. Not ideal, but the `\setcounter` also needs to be updated whenever the first list gets extended, so there's no perfect solution anyway.

**Comment by aisejohan** [comment]
Well, and it only occurs once at the moment!
